sdcm/sct_config.py

```diff
@@ -2428,7 +2428,7 @@ def get_version_based_on_conf(self):  # pylint: disable=too-many-locals
             amis = self.get('ami_id_db_scylla').split()
             region_name = self.region_names[0]
             tags = get_ami_tags(ami_id=amis[0], region_name=region_name)
-            scylla_version = tags.get('scylla_version')
+            scylla_version = tags.get('scylla_version') or tags.get('ScyllaVersion')
```

Comment:
How this change is related to the goal of the commit?

Reply:
I just fixed it while testing on some older Scylla version where we used different tags.

Reply:
it would be nice if it was in it's own separate commit, but it's cosmetic from my POV

Reply:
fixed